docs/manuals/core/config/code_example/data_node_cfg/data-node-config_sql-table.py

```diff
@@ -1,10 +1,27 @@
+# import config start
 from taipy import Config
+# import config end
 
+# mssql example start
 sales_history_cfg = Config.configure_sql_table_data_node(
```

Comment:
The purpose of the python files is also to be able to expose/test the files independently from the doc. Mixing mssql and sqlite in the same script is a bit strange to me. I would create another file. 
Two examples, two files... 
Nothing major though.